app/src/main/res/layout/activity_splash.xml

```diff
@@ -13,7 +13,7 @@
         android:layout_width="wrap_content"
```

Comment:
Should we rename the id `safe_multisig_logo` to `gnosis_safe_logo`?

Reply:
yes, for consistency